### PR TITLE
Add doc for using debugger.

### PR DESCRIPTION
**Pull request by arjunrn**
Add instructions and patch file to use debugger.

### CODE REVIEW

File: docs/reference/development.md [hunk 1]
```diff
@@ -418,3 +418,47 @@ index 0000000..d43c4ea
 +	}
 +}
 ```
+
+### Using a debugger
+Skipper supports plugins and to offer this support it uses the [`plugin`](https://golang.org/pkg/plugin/)
+library. Due to a bug the Go compiler as reported [here](https://github.com/golang/go/issues/23733) a
```
Comment (aryszka) [inline docs/reference/development.md]: "Due to a bug the Go compiler..." -> "Due to a bug in the Go compiler..."

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by arjunrn** [comment]
:+1:

**Pull request merged**